### PR TITLE
add Set#from_yaml

### PR DESCRIPTION
**Pull request by kostya**
Set#to_yaml already exists

### CODE REVIEW

**Comment by RX14** [comment]
Please copy the `Array.new` code exactly.

File: src/yaml/from_yaml.cr [hunk 1]
```diff
@@ -89,6 +89,31 @@ def Array.new(ctx : YAML::ParseContext, node : YAML::Nodes::Node)
   end
 end
 
+def Set.new(ctx : YAML::ParseContext, node : YAML::Nodes::Node)
+  ctx.read_alias(node, self) do |obj|
+    return obj
+  end
+
+  ary = new
```
Comment (wontruefree) [inline src/yaml/from_yaml.cr]: Was this var copied from Array?
should it be something like `new_set` or something?

**Comment by j8r** [comment]
What's the state of this PR? Sounds like mergeable.

**Comment by RX14** [comment]
Sorry, missed the updated PR.

**Comment by kostya** [comment]
i not sure it would work with aliases, but it can fixed separately

**Pull request merged**